plugins/agent-harness-flight-recorder/scripts/evidence_index.py

```
@@ -200,6 +221,705 @@ class Chunk:
+        if trusted_policy is not None and canonical_json(policy) != canonical_json(trusted_policy):
+            raise VaultError("stored relationship policy conflicts")
+        result = query(connection, policy)
+        connection.execute("COMMIT")
+    except Exception:
+        if connection.in_transaction:
+            connection.execute("ROLLBACK")
+        raise
+    finally:
+        connection.close()
+    if _check_existing_database(root / DATABASE_PATH) != identity:
+        raise VaultError("evidence index changed during sealed read")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)